bin/cli.js

```diff
@@ -31,28 +55,26 @@ Object.keys(fileConfig).forEach(configPropName => {
   }
 });
 
-if (!url || flags.help) {
-  if (!flags.help) console.error(getMessageWithPrefix('ERROR', 'NO_URL'));
-  console.error('Usage:');
-  console.error('    pwmetrics http://example.com/');
-  console.error('    pwmetrics http://example.com/ --json  Reports json details to stdout.');
-  console.error('    pwmetrics http://example.com/ --runs=n  Does n runs (eg. 3, 5), and reports the median run\'s numbers.');
-  console.error('    pwmetrics --expectations  Expectations from metrics results. Useful for CI. Config required.');
-  console.error('    pwmetrics --submit  Submit metric results into sheets. Config required.');
-  console.error('    pwmetrics --config  Read config form file.');
-
-  return;
-}
+const write = function(fileName, data) {
```

Comment:
If you wrap this function as a promise you can keep all the process.exits in 1 place, leading to cleaner code in my opinion.
```js
const write = function (fileName, data) {
  return new Promise((resolve, reject) => {
    const path = sysPath.join(process.cwd(), fileName);
    fs.writeFile(path, data, err => {
      if (err) reject(err);
      console.log(getMessageWithPrefix('SUCCESS', 'SAVED_TO_JSON', path));
      resolve();
    });
  });
};
```
This will make the pwmetrics.start also simple. See below.